content/cli/usages.md

```diff
@@ -129,6 +129,7 @@ $ nest start <name> [options]
 | `--path [path]`      | Path to `tsconfig` file. <br/>Alias `-p`                |
 | `--config [path]`    | Path to `nest-cli` configuration file. <br/>Alias `-c`  |
 | `--watch`            | Run in watch mode (live-reload) <br/>Alias `-w`         |
+| `--watchAssets`      | Run in watch non-ts file mode.                          |
```

Comment:
Does `--watchAssets` imply `--watch`?  I.e., do you need to run both and/or should you run them indendently?

Assuming they're independent, I propose the following change to this line:
Run in watch non-ts file mode.
->
Run in watch mode (live-reload), watching non-TS files (assets).  See [Assets](cli/monorepo#assets) for details on assets.

Reply:
Yes, they are running independent. Also a minor change:

Run in watch mode (live-reload), watching non-TS files (assets). See [Assets](cli/monorepo#assets) for _details on assets_.
->
Run in watch mode (live-reload), watching non-TS files (assets). See [Assets](cli/monorepo#assets) for more details.

Reply:
already updated.